test/functional/ui/jumpoptions_spec.lua

```diff
@@ -0,0 +1,106 @@
+-- To run this test:
+--    TEST_FILE=test/functional/example_spec.lua make functionaltest
+
```

Comment:
not relevant for this file

Reply:
Deleted when moving the tests into test/functional/normal/jump_spec.lua.